File: cases/taint/java/model-summary-through-negative/ModelSummaryThroughNegative.java
package dataflowbench.taint;

final class ModelSummaryThroughNegative {
    static String dfb_source() {  // DFB-SOURCE: model-summary-through-input
        return "t";
    }

    static void dfb_sink(String value) { }  // DFB-SINK: model-summary-through-sink

    static void run() {
        dfb_sink(Bridge.hold(dfb_source()));
    }
}

File: cases/taint/swift/model-summary-through-negative/ModelSummaryThroughNegative.swift
func dfb_source() -> String { "tainted" }
func dfb_sink(_ value: String) { print(value) }

func run() {
    let input = dfb_source() // DFB-SOURCE: model-summary-through-input
    dfb_sink(Bridge.hold(input)) // DFB-SINK: model-summary-through-sink
}
